Misc/NEWS.d/next/Library/2017-07-31-13-35-28.bpo-26253.8v_sCs.rst

```diff
@@ -0,0 +1 @@
+Adjustable compression level for tarfile streams.
```

Comment:
```suggestion
Allow adjustable compression level for tarfile streams in 
:function:`tarfile.open`.
```

Reply:
Could you also update the NEWS file to point to #70441 rather than b.p.o -- you may need to regenerate the file, or rename it to the new pattern.

A